force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -82,34 +93,47 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the number is even, False otherwise.
      */
     public static Boolean isEven(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        Integer remainder = Math.mod(num, 2);
+        Boolean isBool = null;
+        if (remainder == 0){
+            isBool = true;
+        } else {
+        isBool = false; 
+    }
+        return isBool; // PASS
     }
 
     /**
-     * Question 5
+     * Question 6
      * Checks if a number is positive.
      * Example: isPositive(2) should return true
      * @param num The number to check.
      * @return true if the number is positive, false otherwise.
      */
     public static Boolean isPositive(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        Boolean isBool = null;
+        if (num > 0){
+            isBool = true;
+        } else {
+        isBool = false; 
+    }
+        return isBool; //PASS
```

Comment:
dont love the variable name is bool. I would just call it result. 

challenge to do this on one line with no conditionals